conference/react-js/src/components/PasswordChangeForm/index.jsx

```diff
@@ -43,6 +43,7 @@ const PasswordChangeForm = ({
       />
       <input
         type="submit"
+        value="Save"
```

Comment:
I think, we should use button type "submit" instead input  type "submit"

Reply:
Button have a strange behaviour in form... I think input is better in this case.